index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MOOC-Capstone Webpage</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="stylesheet" href="capstone.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+                
+</head>
+<body>
+    <section class="home-page">
+        <nav class="home-nav">
+            <ul>
+                <li><i class="fab fa-facebook-f"></i></li>
+                <li><i class="fab fa-twitter"></i></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log Out</a></li>
+            </ul>
+        </nav>
+        <header class="header">
+            <nav>
+                <i class="fas fa-bars hamburger"></i>
+            </nav>
+            <div class="desk-nav">
+                <i class="fas fa-globe">MOOC</i>
+                <ul>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">Blog</a></li>
+                    <li><a href="">News</a></li>
+                    <li><a href="#">Sponsors</a></li>
+                    <li class="call"><a href="#">Call us</a></li>
+                </ul>
+            </div>
+        </header>
+        <div class="mobile-menu">
+            <i class="fas fa-times cancel"></i>
+            <ul class="menu-list">
+                <li><div class="box"><a href="index.html">Home</a></div></li>
+                <li><div class="box"><a href="about.html">About</a></div></li>
+                <li><div class="box"><a href="#">Blog</a></div></li>
+                <li><div class="box"><a href="#">News</a></div></li>
+            </ul>
+        </div> 
+        <div class="head">
+            <div class="header-headings">
+                <h3 class="header-one">An Online Education for All!</h3>
+                <h1 class="header-two">MASSIVE OPEN ONLINE COURSE SUMMIT</h1>
+            </div>
+            <div class="header-para">
+                <p>A Global online conference that discusses all possible approaches to adopt for the Online Education Program.</p>
```

Comment:
- [ ] Please make this change that was suggested currently your popup doesn't match the Figma in terms of colors and the `font`s used. 

![image](https://user-images.githubusercontent.com/18730106/153673055-f622f66f-91d7-4c3e-b0a4-b809aba3cdcd.png)

Reply:
Thank you for your proposal, but this menu page is optional to style. this is how it says in the project.

Reply:
Please, you know the font that is required, I have just put them, check please in my code. I remain with one last chance to submit, so what can I do next?